### DISCUSSION TITLE
How to useC-[ for escape in addition to Esc

### DISCUSSION THREAD

**Discussion by ratish-punnoose** [meow-edit/meow#479] [Q&A]
I would like to use C-[ for escape in addition to the Escape key.

Just to see how to do keybindings, 
I tried doing the following: In the definition for meow-normal-define-key,  (I realize this wouldn't work in Insert mode, but I wanted to see if it would have the same effect).
I added the following line:
```
     '("<escape>" . ignore)
    + '("C-[" . ignore)
```
However, in normal mode the "Escape" key is ignored, but the "C-[" combo is not ignored.
Also: As a side effect, when the line above is added,  M-x does not work.

Can someone explain 1) how to add C-[ as an escape in "Insert" mode  and 2) Why with the added line above M-x does not work?
I'm trying to get my head around meow keybinding.

**Comment by LeoGitHuber** [comment]
Look at following discussion https://github.com/meow-edit/meow/discussions/255 .
